misk/web/tabs/config/src/components/ConfigComponent.tsx

```diff
@@ -0,0 +1,86 @@
+// tslint:disable-next-line:no-var-requires
+const YAML = require("json2yaml")
+import * as React from "react"
+import styled from "styled-components"
+import { IConfigFile } from "../containers/TabContainer"
+
+interface IConfigProps {
+  files: IConfigFile[]
+  status: string
+}
+
+const Container = styled.div`
+  margin-left: 180px;
+  margin-top: 20px;
+`
+
+export default class ConfigComponent extends React.PureComponent<IConfigProps> {
+  indent(spaces: number) {
+    let final = ""
+    for (let i = 0; i < spaces; i++) {
+      final += "\ "
+    }
+    return final
+  }
+
+  toYaml(json: string) {
```

Comment:
neat! this is the kind of thing that really wants unit tests!

Reply:
Absolutely! #372